.circleci/config.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             . venv/bin/activate
             mkdir test-reports
             pip freeze > test-reports/pip_versions.txt
-            pytest -n 2 --cov=ml-agents --cov=ml-agents-envs --cov=gym-unity --cov-report html --junitxml=test-reports/junit.xml -p no:warnings
+            pytest --cov=ml-agents --cov=ml-agents-envs --cov=gym-unity --cov-report html --junitxml=test-reports/junit.xml -p no:warnings
```

Comment:
`pytest-xdist` and PyTorch causes a multiprocessing error to be thrown. Didn't notice a huge increase in testing time though didn't look too deeply into it.